pkg/diff/util.go

```diff
@@ -90,3 +93,18 @@ func rowContainsCols(row map[string]*dbutil.ColumnData, cols []*model.ColumnInfo
 
 	return true
 }
+
+func rowToString(row map[string]*dbutil.ColumnData) string {
+	var s strings.Builder
+	s.WriteString("{ ")
+	for key, val := range row {
+		if val.IsNull {
+			s.WriteString(fmt.Sprintf("%s: IsNull, ", key))
+		} else {
+			s.WriteString(fmt.Sprintf("%s: %s, ", key, val.Data))
```

Comment:
are you planning to convert it to a _JSON_ string? It not accurate, like the ending `,` and may some escaped `:`/`,`, but may not a problem for logging?

Reply:
just for log, it doesn't matter